src/metabase/util/fonts.clj

```diff
@@ -0,0 +1,41 @@
+(ns metabase.util.fonts
+  "font loading functionality."
+  (:require [clojure.string :as str]
+            [clojure.tools.logging :as log]
+            [metabase.util.files :as u.files]))
+
+(def font-path
+  "font resources path as a String."
+  "./resources/frontend_client/app/fonts/")
+
+(defn- normalize-font-dirname
+  "Use a font's directory to derive a Display Name.
+
+  Underscores become spaces, and then split on dashes and take the first word.
+  This is done because of the Lato-v16-latin directory name."
+  [dirname]
+  (-> dirname
+      (str/split #"-")
+      first
+      (str/replace #"_" " ")))
+
+(defn- contains-font-file?
```

Comment:
The function returns the font file (if any), so it might be named `contained-font-file` or something similar.

Reply:
I'll probably wrap in `(boolean ...)` to give the expected output.